<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slice_analysis_Dialog</class>
 <widget class="QDialog" name="Slice_analysis_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="gray">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>550</width>
     <height>370</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="view">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>40</y>
     <width>550</width>
     <height>370</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>180</width>
     <height>260</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>178</width>
      <height>258</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_rows">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rows:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_cols">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Cols:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="row_spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>9</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="col_spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
    <widget class="QPushButton" name="drawBtn">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_percentage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Percentage:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="per_spinBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1180</width>
     <height>430</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="regionTable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>450</y>
     <width>490</width>
     <height>260</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="summaryTable">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>450</y>
     <width>490</width>
     <height>260</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="rgb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>550</width>
     <height>370</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>rgb</zorder>
  <zorder>background</zorder>
  <zorder>view</zorder>
  <zorder>scrollArea</zorder>
  <zorder>regionTable</zorder>
  <zorder>summaryTable</zorder>
  <zorder>gray</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
